docs/esp8266/general.rst

```diff
@@ -66,8 +66,27 @@ avoid allocating too big container objects (lists, dictionaries) and
 buffers. There is also no full-fledged OS to keep track of resources
 and automatically clean them up, so that's the task of a user/user
 application: please be sure to close open files, sockets, etc. as soon
-as possible after use.
-
+as possible after use. The need to close sockets is particularly critical.
+
+Socket instances remain active until they are explicitly closed. This has two
+consequences. Firstly they occupy RAM, so an application which opens sockets
+without closing them will eventually fail. Secondly an open socket can cause
+unexpected behaviour, notably the issuing of ``LmacRxBlk:1`` errors. This can
+occur after a program raises an exception. This includes a ``KeyboardInterrupt``
+resulting from issuing ctrl-C. The REPL prompt appears, but at a later time
+the socket receives data resulting in failure. This manifests as a repeated
+issuing of ``LmacRxBlk:1`` which continues until the ESP8266 is hard reset.
+
+To prevent this, errors should be trapped as follows::
+
+    try:
```

Comment:
This isn't the code which should be recommended/shown as a sample. try/finally is what supposed to be used in such cases.